### ISSUE TITLE
👏 Feature Request: add all installed games to a collection

### ISSUE THREAD

**Issue by pmprog** [itchio/itch#2023]
Dear itch app team, 

I have the app installed on different computers, but my "library" only reflects software I've bought and what's installed locally. Could support be added so that I can see all the items I've installed on other devices too?

 Thanks

**Comment by fasterthanlime** [comment]
That's an interesting feature request! It's way out of scope for v25, but it's cool to have it filed here.

**Comment by pmprog** [comment]
Do you have access to the user's "collections"? You could write a simple approach by creating a private "Installed" collection on their account, then anything you install via the app, check if it's in the collection, and if not, add it; then it'd be visible from the website too.

**Comment by fasterthanlime** [comment]
> Do you have access to the user's "collections"? You could write a simple approach by creating a private "Installed" collection on their account, then anything you install via the app, check if it's in the collection, and if not, add it; then it'd be visible from the website too.

I wasn't thinking of doing it like this!

Would games ever get automatically removed from that collection? Or is it just a collection of all games installed at least once, ever?

**Comment by pmprog** [comment]
I agree that it might not be the best way, it was just an idea that popped into my head that I thought was relatively simple to implement with existing functionality.

I wouldn't mind if it only ever added to the collection, and had to manually delete from the collection.
